### PR TITLE
NiFi 0.3.0 - new formula

### PR DESCRIPTION
**Pull request by apiri**
Creating nifi030 for version 0.3.0 of Apache NiFi (new formula)

### CODE REVIEW

**Comment by apjanke** [comment]
@BrewTestBot test this please

File: nifi030.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Easy, reliable system to process and distribute data"
+  homepage "https://nifi.apache.org"
+  url "https://archive.apache.org/dist/nifi/0.3.0/nifi-0.3.0-bin.tar.gz"
+  sha256 "706a068870a2db85e4fb244a27d302d931c93e358d6001765a5f61b65f1ddfa1"
+
+  conflicts_with "nifi", :because => "Differing versions of the same formula"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
```
Comment (MikeMcQuaid) [inline nifi030.rb]: See https://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1580/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_homebrew_versions_nifi030___new_formula/

**Comment by apjanke** [comment]
Sorry to be a downer (and after I just called for the test build), but it looks like this doesn't meet the [`homebrew-versions` formula guidelines](https://github.com/homebrew/homebrew-versions#acceptable-formulae) any more, as the version is too old. (Not your fault; that's on us for sitting on this so long.)

> Versions formulae must not exceed +/-2 major/minor (not patch) versions from the current stable release.

There've been several major/minor releases since 0.3.0, according to [the Nifi release archives](https://archive.apache.org/dist/nifi/).

```
0.2.1/                                  2015-10-14 21:46    -   
0.3.0/                                  2015-10-14 21:50    -   
0.4.0/                                  2015-12-11 22:25    -   
0.4.1/                                  2015-12-22 15:59    -   
0.5.0/                                  2016-02-17 00:56    -   
0.5.1/                                  2016-02-27 03:06    -   
0.6.0/                                  2016-03-27 03:03    -   
0.6.1/                                  2016-04-16 19:06    -   
0.7.0/                                  2016-07-14 14:05    -   
1.0.0-BETA/                             2016-08-08 21:08    -   
```

Did you need specifically version 0.3.0 of Nifi?

If you need a version of Nifi this old, this is a candidate for [maintaining your own tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) of multiple Nifi versions. Or we'd accept 0.5.x or 0.6.x in `homebrew-versions`, if those met your needs.

**Comment by apiri** [comment]
No worries, was just trying to keep it readily available at the time.  Will roll my own tap, thanks for the link.

**Pull request closed without merging**